src/centreon/common/protocols/actuator/custom/centreonmap.pm

```diff
@@ -171,26 +171,31 @@ sub get_session {
         my $login = { login => $self->{api_username}, password => $self->{api_password} };
         my $post_json = JSON::XS->new->utf8->encode($login);
 
-        my ($content) = $self->{http}->request(
-            method => 'POST',
-            url_path => $self->{url_path} . '/authentication',
-            header => ['Content-type: application/json'],
-            query_form_post => $post_json,
-            warning_status => '', unknown_status => '', critical_status => ''
-        );
+        my @urls = ('/auth/sign-in', '/authentication');
+        my $content;
+        for my $url_path (@urls) {
+            ($content) = $self->{http}->request(
+                method => 'POST',
+                url_path => $self->{url_path} . $url_path,
+                header => ['Content-type: application/json'],
+                query_form_post => $post_json,
+                warning_status => '', unknown_status => '', critical_status => ''
+            );
+            last if ($self->{http}->get_code() == 200);
+        }
 
         if ($self->{http}->get_code() != 200) {
-            $self->{output}->add_option_msg(short_msg => "Authentication error [code: '" . $self->{http}->get_code() . "'] [message: '" . $self->{http}->get_message() . "']");
+            $self->{output}->add_option_msg(short_msg => "All authentication URLs failed");
             $self->{output}->option_exit();
         }
 
         my $decoded = $self->json_decode(content => $content);
-        if (!defined($decoded->{studioSession})) {
-            $self->{output}->add_option_msg(short_msg => 'Cannot studio session');
-            $self->{output}->option_exit();
+        if (!defined($decoded->{jwtToken})) {
+        $self->{output}->add_option_msg(short_msg => 'Cannot studio session');
+        $self->{output}->option_exit();
         }
 
-        $studio_session = $decoded->{studioSession};
+        $studio_session = $decoded->{jwtToken};
```

Comment:
Won't work with the old version (jwtToken for the new one, studioSession for the old one)
Rename $studio_session ($session_token?)